clients/java/src/main/java/io/camunda/client/impl/fetch/ClusterVariableGetRequestImpl.java

```diff
@@ -0,0 +1,86 @@
+/*
+ * Copyright © 2017 camunda services GmbH (info@camunda.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.camunda.client.impl.fetch;
+
+import io.camunda.client.api.CamundaFuture;
+import io.camunda.client.api.fetch.ClusterVariableGetRequest;
+import io.camunda.client.api.search.enums.ClusterVariableScope;
+import io.camunda.client.api.search.response.ClusterVariable;
+import io.camunda.client.impl.command.ArgumentUtil;
+import io.camunda.client.impl.http.HttpCamundaFuture;
+import io.camunda.client.impl.http.HttpClient;
+import io.camunda.client.impl.search.response.ClusterVariableImpl;
+import io.camunda.client.protocol.rest.ClusterVariableResult;
+import java.time.Duration;
+import java.util.concurrent.TimeUnit;
+import org.apache.hc.client5.http.config.RequestConfig;
+
+public class ClusterVariableGetRequestImpl implements ClusterVariableGetRequest {
+
+  private final HttpClient httpClient;
+  private final RequestConfig.Builder httpRequestConfig;
+  private String name;
+  private ClusterVariableScope scope;
+  private String tenantId;
+
+  public ClusterVariableGetRequestImpl(final HttpClient httpClient) {
+    this.httpClient = httpClient;
+    httpRequestConfig = httpClient.newRequestConfig();
+  }
+
+  @Override
+  public ClusterVariableGetRequest requestTimeout(final Duration requestTimeout) {
+    httpRequestConfig.setResponseTimeout(requestTimeout.toMillis(), TimeUnit.MILLISECONDS);
+    return this;
+  }
+
+  @Override
+  public CamundaFuture<ClusterVariable> send() {
+    final HttpCamundaFuture<ClusterVariable> result = new HttpCamundaFuture<>();
+    ArgumentUtil.ensureNotNullNorEmpty("name", name);
+    ArgumentUtil.ensureNotNull("scope", scope);
+    ArgumentUtil.ensureNotNullIf("tenantId", ClusterVariableScope.TENANT.equals(scope), tenantId);
```

Comment:
💭 We can do the validations directly in the methods that receive the input. That also removes the need for the new `ensureNotNullIf`, to be honest. This can be applied to the other request implementations as well.